src/main/java/tech/ypsilon/bbbot/database/codecs/VerificationCodec.java

```diff
@@ -47,20 +31,20 @@ private static String generateVerificationCode() {
         return builder.toString();
     }
 
-    public static VerificationCodec insert(Long userId, String studentCode) {
-        VerificationCodec verificationCodec = new VerificationCodec(new ObjectId(), userId, studentCode,
+    public static VerificationDocument insert(Long userId, String studentCode) {
+        VerificationDocument verificationCodec = new VerificationDocument(new ObjectId(), userId, studentCode,
                 generateVerificationCode(), false, Date.from(Instant.now()));
 
         getCollection().insertOne(verificationCodec);
         return verificationCodec;
     }
 
-    private static MongoCollection<VerificationCodec> getCollection() {
-        return MongoController.getInstance().getCollection(MONGO_COLLECTION, VerificationCodec.class);
+    private static MongoCollection<VerificationDocument> getCollection() {
+        return MongoController.getInstance().getCollection(MONGO_COLLECTION, VerificationDocument.class);
     }
 
     @Override
-    public VerificationCodec decode(BsonReader bsonReader, DecoderContext decoderContext) {
```

Comment:
Auch wenn da vllt Proggen durchschimmert. Hier macht es sinn in encode und decode Konstanten zu verwenden für die Bezeichner. Das macht das ganze im nachhinein einfacher nachzuvollziehen und die Gefahr, dass man nen typo hat ist damit auch eliminiert. Also als Verbesserung was du noch ändern kannst

Reply:
Jo danke für den Hinweis (d63611d)